model/histogram/float_histogram.go

```diff
@@ -341,12 +341,12 @@ func (h *FloatHistogram) Equals(h2 *FloatHistogram) bool {
 // Size returns the total size of the FloatHistogram, which includes the size of the pointer
 // to FloatHistogram, all its fields, and all elements contained in slices.
 // NOTE: this is only valid for 64 bit architectures.
-func (fh *FloatHistogram) Size() int {
```

Comment:
Would it make more sense to use fh everywhere?

Reply:
Yes, we can rename `h` to `fh`, but in that case, we would need to modify 23 function declarations:
<img width="635" alt="image" src="https://github.com/prometheus/prometheus/assets/3228886/7d67ba1a-d447-4a5b-adb6-070bfde98772">

So, I chose to rename only one declaration instead of 23.